composer.json

```diff
@@ -50,9 +50,10 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-main": "2.0.3-dev"
+            "dev-main": "10.1.0-dev"
```

Comment:
Why not 10.0.0-dev ? There is to my knowledge now 10.0.0 version released yet.

Reply:
You are right, my fault. It should be `9.0.0-dev`, because it is compatible for TYPO3V9 and TYPO3V10.